development/build/scripts.js

```diff
@@ -635,12 +516,16 @@ function createFactoredBuild({
     const environment = getEnvironment({ buildTarget });
     const config = await getConfig(buildType, environment);
     const { variables, activeBuild } = config;
-    await setEnvironmentVariables({
-      buildTarget,
+    setEnvironmentVariables({
+      isDevBuild: reloadOnChange,
+      isTestBuild: isTestBuild(buildTarget),
+      buildName: getBuildName({
+        environment,
+        buildType,
+      }),
```

Comment:
The three functions used here, `isDevBuild` (`reloadOnChange` holds the result of calling `isDevBuild`), `isTestBuild`, and `getBuildName` are needed by other parts of the build process, but not the webpack build is (currently) always a dev build and never a test build.

These functions were previously called from within `setEnvironmentVariables`, but in the webpack PR I don't want to import `build/utils.js` (where these functions live) because `build/utils.js` is slow to import. I _could_ further refactor things to pull out `isTestBuild` and `isDevBuild` into a smaller file that is less "expensive" to import, and I may do that in the future, but for now I just hope that this is okay as is.